examples/draganddrop/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>solid-ui UI.widgets.draganddrop examples page</title>
+    <script type="text/javascript" src="../../lib/webpack-bundle.js"></script>
+    <script>
+      const $ = document.querySelector.bind(document)
+      function showSource(widgetName) {
+        $(`#viewSource-${widgetName}`).innerHTML = $(`#script-${widgetName}`)
+          .innerText.replace(/&/g, '&amp;')
+          .replace(/</g, '&lt;')
+          .replace(/>/g, '&gt;')
+          .replace(/"/g, '&quot;')
+          .replace(/'/g, '&#039;')
+      }
+    </script>
+  </head>
+  <body>
+    <h2 id="makeDropTarget"><a href="#makeDropTarget">makeDropTarget</a></h2>
+    <p id="drag" draggable="true">This element is draggable.</p>
+    <script id="script-makeDropTarget">
+      const ele = document.querySelector('#drag')
```

Comment:
This makes it very confusing; you're using one and the same element for both drag and drop. Please create a second element next to the "This element is draggable" one, that says "This is a drop target", then make that a drop target.

Reply:
aww yes sorry. this was not complete.  this was me playing around and running into the error where I couldn't run the build-lib on windows to serve my example.  I haven't revisited yet.

Reply:
Regarding the branch coverage, due to the fact we can't create a DragEvent the coverage will not be good for this file.  I will make a comment about this in the file.  The reason this is the case is that you need to use exactly DragEvent to create the dataTransfer property, otherwise just creating new Event('dragover', ...) does not create this property at least in Jest.